rest_framework_latex/renderers.py

```diff
@@ -49,7 +49,7 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
         """
         # Get latex
         tex = super(LatexRenderer, self).render(
-            data, accepted_media_type, renderer_context)
+            {'data': data}, accepted_media_type, renderer_context)
```

Comment:
Personally, I'd be keen to not change the expected behaviour when a dictionary is passed - this change will wrap:

```python
data = {
  'key': 'val',
}
```

into

```python
data = {
  'data': {
    'key': 'value'
  }
}
```

which would break existing applications depending on the data dictionary being passed verbatim in the template.